source/tools/GenContainer.rst

```diff
@@ -16,39 +16,95 @@ A container is an encapsulation of multiple components as depicted in the follow
 Image Types
 ===========
 
-Normal
-^^^^^^
+1. Normal
+^^^^^^^^^
 
-This type of container contains a single image in it. Image format can be ELF, Multiboot, Multiboot-2, UEFI-PI FV, or PE32.
+This type of container contains a single component in it. Component format can be ELF, UEFI-PI FV, or PE32.
 
-Classic
-^^^^^^^
+ - example command::
 
-This image type is used to boot a typical Linux image. This image format combines a kernel command line and the kernel image.
-Optionally more files like an initrd, ACPI tables, firmware files could be added. 
+    python GenContainer.py create -t NORMAL -cl UEFI:UefiPld.fd
+
+2. Classic Linux
+^^^^^^^^^^^^^^^^
+
+This container type is used to boot a typical Linux operating system. This container is made up of atleast two
+components - the kernel command line, and kernel image.
+
+**SBL expects the components inside the container in the following order: kernel command line, kernel image,
+and the optional initrd.**
```

Comment:
SBL expects the components inside the container in the following order: kernel command line, kernel image,
and the optional initrd --> SBL expects the components inside the container in the following order: kernel command line, kernel image and an optional initrd.